src/lithium/reducer.py

```diff
@@ -1166,6 +1199,38 @@ def tryArgumentsAsGlobals(roundNum, testcase, interesting, tempFilename):  # pyl
         return numMovedArguments, testcase
 
 
+class CollapseEmptyBraces(Minimize):
+    """ Perform standard line based reduction but collapse empty braces at the end of each round
+    This ensures that empty braces are reduced in a single pass of the reduction strategy
+
+    Example:
+        // Original
+        function foo() {
+        }
+
+        // Post-processed
+        function foo() { }
+    """
+    name = "minimize-collapse-brace"
+
+    @staticmethod
+    def post_round_cb(testcase):  # pylint: disable=missing-docstring,missing-return-doc,missing-return-type-doc
```

Comment:
I feel we are standardising docstrings to be of the following format [similar to here in Lithium](https://github.com/MozillaSecurity/lithium/blob/master/src/lithium/interestingness/crashes.py#L22) - here is a suggestion:

```
    """Callback function to be performed after the round ends.
    Args:
        testcase (Testcase): Testcase to be reduced.
    Returns:
        bool: True if callback was performed successfully, False otherwise.
    """
```

Reply:
The pylint error ignores can then be removed.